desc/equilibrium/equilibrium.py

```diff
@@ -1153,6 +1153,44 @@ def map_coordinates(  # noqa: C901
             **kwargs,
         )
 
+    def rtz_grid(
+        self, radial, poloidal, toroidal, coordinates, period, jitable=True, **kwargs
+    ):
+        """Return DESC coordinate grid from given coordinates.
+
+        Create a meshgrid from the given coordinates, and return the
```

Comment:
Could use a bit more explanation here, maybe an example of how/when its used/needed

Reply:
also might be clearer as `get_rtz_grid`, right now it sounds more like an attribute property.

Also, does it need to be a method? Is this something the user needs to think about? Or is the semi-private function in utils enough?

Reply:
I made this an equilibrium method to avoid circular import errors when importing EffectiveRipple objective that occur when `rtz_grid` is declared as a function in equilibium/coords.py.

I think putting it in compute utils instead of coords would also avoid circular import